src/projects/apps/designer/explorer.md

```diff
@@ -0,0 +1,11 @@
+# Explorer 
+
+The Explorer tab **shows all the components added to an application page**. 
+
+The app components are arranged in the order you want. Each page starts with the Root which represents the base page, and then subsequent components that are added to the page will appear below the root. You have the option to move each app component, just simply select it and move it to the new place. You can also drag components directly from the components tab onto the explorer for more precise placement.
```

Comment:
The app components aren't `arranged in the order you want`, they're in the order they are on the page. I think we should be consistent about using the word `drag` rather than `move` which is a bit vague.